l10n_it_intrastat_statement/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': "ITA - Dichiarazione Intrastat",
-    'version': '10.0.1.0.1',
+    'version': '10.0.1.0.2',
```

Comment:
Non è necessario modificare la versione in quanto verrà poi fatto da @OCA-git-bot